### PR TITLE
Create Environments Beta page

### PR DESCRIPTION
**Pull request by dbreseman**
Planned release 14 October
Final review starts 7 October

### CODE REVIEW

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+You can perform the following actions on this tab:
+
+* Use the **Search** field to search the deployment packages by name or version
+* Click **Upload a Package** to upload a deployment package from your local device
+* Click **Create a Package** to create a deployment package based on a specific branch
+
+You can also see information and perform actions on the existing deployment packages. For each deployment package, the table contains the following details:
+
+* Deployment package name
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/deployment/mendix-cloud-deploy/environments-new.md:93]: Also status & deployment label

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+* For the deployment package, you can view the package name, version, and runtime. To launch the **Promote a  Package** wizard and promote the package to another environment, click **Promote**.
+
+    {{< figure src="/attachments/deployment/mendix-cloud-deploy/environments-beta/promote-dialog.png" alt="" >}}
+
+### Deployment Packages Tab{#deployment-packages-tab}
+
+In the **Deployment Packages** tab, you can view information about your deployment packages.
+
+{{< figure src="/attachments/deployment/mendix-cloud-deploy/environments-beta/deployment-packages-tab.png" alt="" >}}
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/deployment/mendix-cloud-deploy/environments-new.md:82]: remove the blue dot

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+You can also see information and perform actions on the existing deployment packages. For each deployment package, the table contains the following details:
+
+* Deployment package name
+* Version of the deployment package
+* Date that the deployment package was uploaded
+* Uploader account
+* Expiry date of the deployment package, if one is set
+    * Currently deployed packages do not expire. Among any non-deployed packages, the five most recently deployed packages do not expire.
+    * Expiry dates apply only to packages that do not meet either of two non-expiry criteria outlined above. These packages are set to expire and be deleted within 15 days, unless they are deployed before that date.
+
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/deployment/mendix-cloud-deploy/environments-new.md:101]: deploy icon to deploy

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+Operations performed on a customer's behalf through an authorized Mendix Support ticket are not logged in the activity log.
+{{% /alert %}}
+
+If you are the [Technical Contact](/developerportal/general/app-roles/#technical-contact), you can download a copy of the activity log by following these steps:
+
+1. Click **Download to CSV**. (This button is only visible if you are the Technical Contact.)
+2. Select the period for which you want to download log entries.
+3. Click **Download**.
+
+## Cloud Settings{#cloud-settings}
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/deployment/mendix-cloud-deploy/environments-new.md:131]: Environment settings missing; available for technical contact only

Comment (katarzyna-koltun-mx) [reply content/en/docs/deployment/mendix-cloud-deploy/environments-new.md:131]: Maybe a new section called Settings for Technical Contact

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+
+Team members who have a role with **Cloud Access** can see which services are enabled or disabled.
+
+Once a service has been enabled for an app, Technical Contacts can selectively enable it for each environment. For more information, see the [Services](/developerportal/deploy/environments-details/#services) section of the *Environment Details* page.
+
+## Deploy Wizard{#deploy-wizard}
+
+If you click **Deploy** ({{< icon name="deploy" >}}) from the **Overview** tab or the **Deployment Packages** tab, this launches the Deploy wizard. 
+
+### Step 1: Select Package
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/deployment/mendix-cloud-deploy/environments-new.md]: Replace with text

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+
+{{< figure src="/attachments/deployment/mendix-cloud-deploy/environments-beta/1-select-package.png" alt="" >}}
+
+### Step 2: Configure Environment
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/deployment/mendix-cloud-deploy/environments-new.md]: live value=after deployment

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+
+{{< figure src="/attachments/deployment/mendix-cloud-deploy/environments-beta/2-configure-environment.png" alt="" >}}
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/deployment/mendix-cloud-deploy/environments-new.md]: sliding panel

Comment (katarzyna-koltun-mx) [reply content/en/docs/deployment/mendix-cloud-deploy/environments-new.md]: Mask=hide value

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+
+{{< figure src="/attachments/deployment/mendix-cloud-deploy/environments-beta/edit-configuration.png" alt="" >}}
+
+### Step 3: Backup
+
+{{< figure src="/attachments/deployment/mendix-cloud-deploy/environments-beta/3-backup.png" alt="" >}}
+
+### Step 4: Deploy
+
+{{< figure src="/attachments/deployment/mendix-cloud-deploy/environments-beta/4-deploy.png" alt="" >}}
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/deployment/mendix-cloud-deploy/environments-new.md]: Now says Summary not Overview

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+weight: 2
+description: "Describes the new Environments page and deployment flow"
+---
+
+{{% alert color="warning" %}}
+This feature is in Public Beta. For more information, see [Beta Releases](/releasenotes/beta-features/).
+{{% /alert %}}
+
+{{% alert color="info" %}}
+If your app is running on Cloud Foundry, you can use the **New Theme** toggle to switch between this new UI and the [legacy UI](/developerportal/deploy/environments/). If your app is running on Kubernetes, the new UI displays by default.
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/deployment/mendix-cloud-deploy/environments-new.md]: Don't mention cloud foundry, say "while the feature is in beta". On Kubernetes this feature is enabled and can't be disabled

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md [hunk 10]
```diff
@@ -0,0 +1,203 @@
+---
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/deployment/mendix-cloud-deploy/environments-new.md:1]: move above Pipelines in the nav tree

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md (hunk 9 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/deployment/mendix-cloud-deploy/environments-new.md]: Done. @feddovanede please check the wording.

File: content/en/docs/deployment/mendix-cloud-deploy/environments-new.md (hunk 2 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/deployment/mendix-cloud-deploy/environments-new.md:82]: Done

**Review by feddovanede — COMMENTED** [review]
@katarzyna-koltun-mx I will go through it in detail tomorrow. One thing I see is missing, is the Promote wizard. This one very similar to the Deploy wizard, but it is different. Let's discuss tomorrow.

**Comment by katarzyna-koltun-mx** [comment]
@feddovanede Hi Feddo, I've done the changes that we discussed. Let me know if there's anything else.

**Comment by katarzyna-koltun-mx** [comment]
@feddovanede I've implemented the changes that we discussed, let me know if there's anything else, or if I should publish the PR.

**Pull request merged**